### PR TITLE
Rewrite the logic of calculating liquid tokens by using implementation from core-contracts

### LINKED ISSUE #9: Rewrite the logic to the same as in core-contracts/lockup
That should help us to get rid of bugs, hopefully

### PR DESCRIPTION
**Pull request by telezhnaya**
We've rewritten the solution from Rust to JS here https://github.com/near/near-explorer/blob/master/backend/src/aggregations.js

Put it here

I also had a war with the BN library, I needed to update the version to use the same as in near-explorer, that's why we have some diff in `yarn.lock`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+This is a temporary tool to get the data from the deployed lockup contracts.
+
+## Contributing
+
+To run this tool locally, use:
+```
+npm i -g yarn
+yarn
+/usr/local/bin/node /usr/local/lib/node_modules/yarn/bin/yarn.js run start
```
Comment (frol) [inline README.md]: O_o

```suggestion
yarn
yarn start
```

File: script.js [hunk 2]
```diff
@@ -26,29 +26,26 @@ function prepareAccountId(data) {
...
+    return x === 1 ? f() : defaultValue;
+};
 
 async function viewLockupState(connection, contractId) {
     const result = await connection.provider.sendJsonRpc("query", {
         request_type: "view_state",
         finality: "final",
         account_id: contractId,
-        prefix_base64: "U1RBVEU=",
+        prefix_base64: "",
```
Comment (frol) [inline script.js]: Revert it back

```
$ echo 'U1RBVEU=' | base64 -d
STATE
```

**Review by frol — CHANGES_REQUESTED** [review]
While the code looks fine, it somehow breaks on some contracts:

`ed25519:6paKsDgqeeUQBfBU2V2wyUNGe2T4SZgVyCuB82jqd9nk` (you can lookup implicit accounts by the public key) from https://github.com/near/account-lookup/pull/7 is broken.

Please, check all the other PRs for the examples that caused issues previously and confirm that they run fine.

File: README.md (hunk 1 above)
Comment (telezhnaya) [reply README.md]: Sorry, I copy-pasted generated commands from IDE 😄

**Comment by telezhnaya** [comment]
Looks like I fixed #11, checked with `mike.near` (it looks a little different because includes owners balance, sounds logical) and `timp.near` (it has staking info).
Also checked all other examples: hopefully we cover all the examples now.
Please have a look again

**Review by frol — APPROVED** [review]
It seems to be working fine with my limited testing and reading the code. I hope it is the last time we need to fix it...

**Pull request merged**